readme.md

```diff
@@ -9,23 +9,25 @@ The Launchbox SDK provides a set of functions to interact with Farcaster channel
 To install the Launchbox SDK, you need to include it in your project. Use npm or yarn to install the package:
 
 ```sh
-npm install @airstack/node
+npm install channels-lib
 ```
 
 or using Yarn
 
 ```sh
-yarn add @airstack/node
+yarn install channels-lib
```

Comment:
yarn add channels-lib

Reply:
> yarn add channels-lib

That moment you get a `golang` dev to work on ts project 🤣